coramin/relaxations/iterators.py

```diff
@@ -0,0 +1,69 @@
+import pyomo.environ as pe
+from .relaxations_base import BaseRelaxationData
+
+
+def relaxation_data_objects(block, descend_into=True, active=None, sort=False):
+    """
+    Iterate over all instances of BaseRelaxationData in the block.
+
+    Parameters
+    ----------
+    block: pyomo.core.base.block._BlockData
+        The Block in which to look for relaxations
+    descend_into: bool
+        Whether or not to look for relaxations in sub-blocks
+    active: bool
+        If True, then any relaxations that have been deactivated or live on deactivated blocks will not be returned.
+    sort: bool
+
+    Returns
+    -------
+    relaxations: generator
+        A generator yielding the relaxation objects.
+    """
+    for b in block.component_data_objects(pe.Block, descend_into=descend_into, active=active, sort=sort):
+        if isinstance(b, BaseRelaxationData):
+            yield b
```

Comment:
What is the expected behavior if this is called on a relaxation data object?  Should it return that block?  As it is currently implemented, it will not.  If you want that behavior (that `block` will be returned), substitute `component_data_objects` with `block_data_objects`.

Reply:
I was intending the behavior to be more like `component_data_objects` and less like `block_data_objects`.